docs/dag-run.rst

```diff
@@ -0,0 +1,184 @@
+``DAG Run`` entry in backend.
+
+**Note**: If you run a DAG on a schedule_interval of one day, the run stamped 2020-01-01 
+will be triggered soon after 2020-01-01T23:59. In other words, the job instance is 
+started once the period it covers has ended.  The execution_date passed in the dag 
+will also be 2020-01-01.
+
+The first ``DAG Run`` is created based on the minimum ``start_date`` for the tasks in your DAG. 
+Subsequent ``DAG Runs`` are created by the scheduler process, based on your DAG’s ``schedule_interval``, 
+sequentially. If your start_date is 2020-01-01 and schedule_interval is @daily the first run 
+will be created on 2020-01-02 i.e. after your start date has passed.
+
+Re-run DAG
+''''''''''
+
+Catchup
+-------
+
+An Airflow DAG with a ``start_date``, possibly an ``end_date``, and a ``schedule_interval`` defines a 
+series of intervals which the scheduler turn into individual DAG Runs and execute. A key capability 
+of Airflow is that these DAG Runs are atomic and idempotent items. The scheduler, by default, will
+kick off a DAG Run for any interval that has not been run (or has been cleared). This concept is called Catchup.
+
+If your DAG is written to handle its own catchup (i.e. not limited to the interval, but instead to ``Now`` for instance.), 
+then you will want to turn catchup off. This can be done by setting ``catchup = False`` in DAG  or ``catchup_by_default = False``
+in configuration file. When turned off, the scheduler creates a DAG run only for the latest interval.
+
+.. code:: python
+
+    """
+    Code that goes along with the Airflow tutorial located at:
+    https://github.com/apache/airflow/blob/master/airflow/example_dags/tutorial.py
+    """
+    from airflow import DAG
+    from airflow.operators.bash_operator import BashOperator
+    from datetime import datetime, timedelta
+
+
+    default_args = {
+        'owner': 'Airflow',
+        'depends_on_past': False,
+        'start_date': datetime(2015, 12, 1),
+        'email': ['airflow@example.com'],
+        'email_on_failure': False,
+        'email_on_retry': False,
+        'retries': 1,
+        'retry_delay': timedelta(minutes=5)
+    }
+
+    dag = DAG(
+        'tutorial',
+        default_args=default_args,
+        description='A simple tutorial DAG',
+        schedule_interval='@daily',
+        catchup=False)
+
+In the example above, if the DAG is picked up by the scheduler daemon on 2016-01-02 at 6 AM, 
+(or from the command line), a single DAG Run will be created, with an `execution_date` of 2016-01-01, 
+and the next one will be created just after midnight on the morning of 2016-01-03 with an execution date of 2016-01-02.
+
+If the `dag.catchup` value had been True instead, the scheduler would have created a DAG Run 
+for each completed interval between 2015-12-01 and 2016-01-02 (but not yet one for 2016-01-02, 
+as that interval hasn’t completed) and the scheduler will execute them sequentially. This behavior 
```

Comment:
```suggestion
as that interval hasn’t completed) and the scheduler will execute them sequentially. 

This behavior 
```
This is a very important summary sentence, which should be included in a separate paragraph.